src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -3943,6 +4087,67 @@ export class ResultsViewPageStore {
         },
     });
 
+    readonly getStructuralVariantPutativeDriverInfo = remoteData({
+        await: () => {
+            const toAwait = [];
```

Comment:
@adamabeshouse @kalletlak i'm just thinkng about these dynamic awaits.  i know we've long used them.  but do they actually work?  when does await get called (only at setup time or is it reactive?). just kind of curious.  it seems possible to me that we think it's working but it actually doesn't do what we think.  it would have to re-evaluate what it needs to await every time something changes, but if one of the conditions evaluates false initially, it won't actually be registered with its change event.

Reply:
@alisman in my opinion the fact that we're not seeing bugs is proof that it does work, but we could do a more purposive test if necessary. 

In theory it does work - `await` is not `@computed` but it is referenced by `@computed status`, which means that any changed observables in `await` will trigger recomputation of `status` and `await`. `driverAnnotationSettings` is an observable object so even if the conditions evaluate false they will trigger a recomputation when they change. The only gotcha is to make sure in `invoke` that you're not expecting a promise to be ready that wasn't actually awaited.